src/webapp/components/user-list-table/UserListTable.tsx

```diff
@@ -55,10 +57,13 @@ function convertActionToOrgUnitType(action: ActionType): SaveUserOrgUnitOptions[
             return "output";
         case "assign_to_org_units_search":
             return "search";
-        case "copy_in_user":
-        case "disable":
-        case "enable":
-        case "remove":
+        // Note for reviewer: Commented these lines just to keep track of them and because linter was complaining about return statement
+        // If you give me the ok, I would restrict the ActionType to OrgUnitActionType
+        // case "copy_in_user":
+        // case "disable":
+        // case "enable":
+        // case "remove":
```

Comment:
I didn't have any warning with eslint, but yeah restricting to OrgUnitActionType sounds a good idea.